edweiss-app/components/memento/CardDisplayComponent.tsx

```diff
@@ -51,7 +51,6 @@ const CardDisplayComponent: ReactComponent<{
                 }} >
 
                 <TText bold mb="sm">Answer:</TText>
-                {/*<TText>{isAnswerVisible ? card.answer : 'Click to reveal the answer'}</TText>*/}
```

Comment:
You maybe can delete this commented line

Reply:
Ah yes